src/qt/walletmodel.cpp

```diff
@@ -313,6 +313,21 @@ bool WalletModel::backupWallet(const QString &filename)
     return BackupWallet(*wallet, filename.toLocal8Bit().data());
 }
 
+// getStakeForCharityPercent returns the wallet int setting nStakeForCharityPercent.
```

Comment:
When you activate stake for charity, the wallet will watch for new blocks coming in from you or the network. After a new block is found and accepted by the network the wallet will check if you have any PoS blocks that have matured. At this point the wallet will calculate your reward and send a percentage of that to your address of choice.  Since the user can choose the percentage, getStakeForCharityPercent just retreives the current percentage.